docs/pages/access-controls/login-rules/reference.mdx

```diff
@@ -585,6 +586,82 @@ Expression | Result
 `strings.replaceall(set("user-name"), "-", "_")` | `("user_name")`
 `strings.replaceall(set("user-alice", "user-bob"), "user-", "")` | `("alice", "bob")`
 
+### `email.local`
+
+<Admonition type="note">
+The `email.local` helper was introduced in Teleport v13.3.0, all Auth servers
+must be running this version or greater before it can be used.
+</Admonition>
+
+#### Signature
+
+```go
+func email.local(input set) set
+```
+
+#### Description
+
+`email.local` maps each email in the input set to its local part.
+
+#### Arguments
+
+Argument | Type | Description
+-------- | ---- | -----------
+`input` | `set` | set of input emails |
+
+#### Returns
+
+Type | Description
+---- | ----------
+`set`  | a copy of `input` where each email has been converted to its local part.
+
+#### Examples
+
+Expression | Result
+------- | ------
+`email.local(set("alice@example.com"))` | `("alice")`
+`email.local(set("Alice <alice@example.com>"))` | `("alice")`
+
+### `regexp.replace`
+
+<Admonition type="note">
+The `regexp.replace` helper was introduced in Teleport v13.3.0, all Auth servers
+must be running this version or greater before it can be used.
```

Comment:
```suggestion
The `regexp.replace` helper was introduced in Teleport v13.3.0. All Auth Service instances
must be running this version or greater before it can be used.
```